packages/sitecore-jss/src/utils/utils.ts

```diff
@@ -75,3 +76,32 @@ export const isTimeoutError = (error: unknown) => {
     (error as Error).name === 'AbortError'
   );
 };
+
+const convertToRegex = (pattern: string) => {
+  return pattern.replace('.', '\\.').replace(/\*/g, '.*');
+};
+
+export const enforceCors = (
+  req: IncomingMessage,
+  res: OutgoingMessage,
+  allowedOrigins?: string[]
+): boolean => {
+  const defaultAllowedOrigins = process.env.API_ALLOWED_ORIGINS
```

Comment:
> If not, would be nice to mention EDITING in a name of the variable, since this name is so general and doesn't explain what exactly should be defined.

I want the "enforceCors" method to be generic enough so it can be used for all endpoints when needed, not only editing ones.
The logic is two-step. Default behavior is checking the request against the env value only. Then additional allowed values can be provided via allowedOrigins parameter. This how editing-specific origins are passed in editing middleware handlers.
With this goal, it's not right to call the env variable EDITING_CORS_ORIGINS, but I'm not a big fan of API_ALLOWED_ORIGINS either. What name would you have picked?

Reply:
Something like JSS_ALLOWED_ORIGINS perhaps?

Reply:
Yeah, looks good as we discussed